index.html

```diff
@@ -1824,7 +1844,7 @@ <h2 id="docconformance">
           </tr>
           <tr id="progress" tabindex="-1">
             <td>
-              <code><a data-xref-for="HTMLElement">progress</a></code>
+              <code><a>progress</a></code>
```

Comment:
If this is a  reference to the `<progress>` element, may I suggest: 

```suggestion
              [^progress^]
```